File: Assignment4/district.py
import pandas as pd

df = pd.read_csv('data/districts.csv')

df[['municipality500','municipality1999','municipality9999','municipality10000']] = df.municipality_info.str.replace('[','').str.replace(']','').str.split(',').tolist()
df = df.drop('municipality_info',axis=1)

df[['unemployment_rate95','unemployment_rate96']] = df.unemployment_rate.str.replace('[','').str.replace(']','').str.split(',').tolist()
df = df.drop('unemployment_rate',axis=1)

df[['commited_crimes95','commited_crimes96']] = df.commited_crimes.str.replace('[','').str.replace(']','').str.split(',').tolist()
df = df.drop('commited_crimes',axis=1)

df.to_csv('district_py.csv',index=False)

File: Assignment4/district.R
library(dplyr)
library(tidyr)
df = read.csv('data/districts.csv')

df = as.data.frame(apply(df,2, function(y) gsub('\\[|\\]','',y)))

tidy_df = df %>%
  separate(municipality_info,
           into = c('municipality500','municipality1999','municipality9999','municipality10000'),
           sep=',') %>%
  separate(unemployment_rate,
           into = c('unemployment_rate95','unemployment_rate96'),
           sep=',') %>%
  separate(commited_crimes,
           into = c('commited_crimes95','commited_crimes96'),
           sep=',')


write.csv(tidy_df,'district_r.csv')
